### PR TITLE
[modular][ready] adds a uniform in the loadout for the superstar cops

### PR DESCRIPTION
**Pull request by thestubborn**
using my conceptualization, i've brought gifts for us drunk, homeless, slightly homoerotic Superstar Cops of the Apocalypse

this adds some truly terrible clothing, why any sane man would wear them escapes you, but by god do they scream disco, and confidence

### CODE REVIEW

**Comment by Gandalf2k15** [comment]
Yo something tells me the accessories.dm isn't included in the DME

**Comment by Gandalf2k15** [comment]
image

**Comment by thestubborn** [comment]
ahh, i dont think i added anything to that, but sure, gimme a sec

File: modular_skyrat/modules/customization/modules/clothing/under/accessories.dm [hunk 1]
```diff
@@ -0,0 +1,8 @@
+
```
Comment (Gandalf2k15) [inline modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: why this

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: to get it to work, you wanted it ticked so I ticked it, made it white space, I was gonna add the tie as an accessory before I realised doing it as a tie was better

Comment (Gandalf2k15) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: ya, remove the file and include :)

Comment (thestubborn) [reply modular_skyrat/modules/customization/modules/clothing/under/accessories.dm]: done

**Comment by SpaceVampire** [comment]
Could we uh. See some photos of them in the actual PR?

**Comment by thestubborn** [comment]
ok then dork
image
jacket
image
undersuit
image
tie
image
feet

**Comment by GoldenAlpharex** [comment]
Maybe follow the PR format too? Description, Why it's good for the game? and Changelog.

**Comment by thestubborn** [comment]
> 
> 
> Maybe follow the PR format too? Description, Why it's good for the game? and Changelog.

i've made a few prs without doing it and I feel kinda dedicated to the bit now

**Comment by Draganfrukts** [comment]
> i've made a few prs without doing it and I feel kinda dedicated to the bit now

Remember that the changelog section actually makes your changes show up in the in-game pop up when you join

**Comment by thestubborn** [comment]
> 
> 
> > i've made a few prs without doing it and I feel kinda dedicated to the bit now
> 
> Remember that the changelog section actually makes your changes show up in the in-game pop up when you join

people read that?

**Comment by Helianthus00** [comment]
Yet another PR taken with no credit to the original creators. Completely shameless of you. Even your description makes it appear as if you created this. 

The original code was written by Surrealaser and the sprites done by d4n0w4r. I'd very much appreciate if you all could credit where you take things from in the future, at the very least. https://github.com/TheSwain/Fulpstation/pull/306

**Comment by thestubborn** [comment]
> 
> 
> Yet another PR taken with no credit to the original creators. Completely shameless of you. Even your description makes it appear as if you created this.
> 
> The original code was written by Surrealaser and the sprites done by d4n0w4r. I'd very much appreciate if you all could credit where you take things from in the future, at the very least. [TheSwain/Fulpstation#306](https://github.com/TheSwain/Fulpstation/pull/306)

that wasn't my intent, I was told off after this and actually learned my lesson, all apologies

**Pull request merged**